client/components/NewStory.tsx

```diff
@@ -37,8 +38,9 @@ export default function NewStory() {
   }
 
   async function handleSuprise() {
-    const readingLevels = Object.keys(readingLevelOptions);
-    const randomReadingLevel = readingLevels[Math.floor(Math.random() * readingLevels.length)];
+    if (!selectedProfile) return;
+    const readingLevels = selectedProfile.readingLevel;
```

Comment:
this var name should not be plural

Reply:
I made this plural though just to be cautious as readingLevel is within profiles, etc